bundle/Resources/views/pagelayout.html.twig

```diff
@@ -1,11 +1,14 @@
+{# content \Netgen\IbexaSiteApi\API\Values\Content #}
+{# location \Netgen\IbexaSiteApi\API\Values\Location #}
+
 <!doctype html>
 <html lang="{{ app.request.locale|replace({'_': '-'}) }}">
 <head>
     <meta charset="utf-8">
-    <title>{{ content.name|default( 'Home' ) }}</title>
+    <title>{{ content.name }}</title>
     <meta name="generator" content="Netgen Site API"/>
-    {% if content is defined and content.mainLocation is not null %}
-        <link rel="canonical" href="{{ ibexa_path( content.mainLocation ) }}" />
+    {% if location.url is not null %}
```

Comment:
Pagelayout is used when there's no content and location defined, in custom modules and such. So the `is defined` check would still be needed.

Reply:
Fixed in https://github.com/netgen/ibexa-site-api/pull/11/commits/59eedfdf273936da2cdc10397bc92ca1c13ae898.